server/settings/publish_plugins.py

```diff
@@ -797,8 +797,8 @@ class PublishPuginsModel(BaseSettingsModel):
         default_factory=ValidateBaseModel,
         title="Validate Version"
     )
-    ValidateContainers: ValidateContainersModel = SettingsField(
-        default_factory=ValidateContainersModel,
+    ValidateOutdatedContainers: ValidateOutdatedContainersModel = SettingsField(
+        default_factory=ValidateOutdatedContainersModel,
         title="Validate Containers"
```

Comment:
I wonder if the label should be updated.
```suggestion
        title="Validate Outdated Containers"
```